meshroom/multiview.py

```diff
@@ -19,6 +19,8 @@
     '.gif',
     # hdr:
     '.hdr', '.rgbe',
+    # heif
+    '.heic', '.heif', '.avif'
```

Comment:
Missing comma:
```suggestion
    '.heic', '.heif', '.avif',
```

Reply:
Thanks, reset, added the comma, and recommitted!